skatepark/sass/block-patterns/_mixed-media-in-container.scss

```diff
@@ -1,5 +1,6 @@
-.skatepark-mixed-media-in-container {
-	hr, p, h1, h2, h3, h4, h5, h6 { // Shrink default vertical margins for all the elements in this pattern
+.skatepark-mixed-media-in-container,
+.columns-in-container {
+	hr, p, h1, h2, h3, h4, h5, h6 { // Shrink default vertical margins for all the elements in these patterns
```

Comment:
I'm not very keen on this. It makes it harder users to reproduce the same layouts without patterns and seems like it would create inconsistency between different parts of the site. Also, there are suggestions to add spacing back to this pattern under the image. If we didn't have these rules then I think we wouldn't need to add the extra spacing